generators/express/templates/ansible/provision.yaml.ejs

```diff
@@ -50,7 +50,7 @@
           client_max_body_size 25M;
 
           location / {
-              proxy_pass http://localhost:3000/;
+              proxy_pass http://localhost:4000/;
```

Comment:
And this means we should either put back `3000` here or set the `PORT` env variable in the ansible configuration.

Reply:
Not sure I'm following you between your 2 comments. Assuming that I provide a `PORT` how this would work being `3000` here ? We do agree that I would need to : 

1) have the fallback port in express to be `3000`
2) follow the same logic in ansible, having either the provided `PORT` or defaulting back to `3000`

Reply:
What I meant is that we either have to : 
 - revert to 3000 here like you did
 - keep 4000 et add a `PORT: '4000'` to the env map in the ansible config.